java/code/src/com/redhat/rhn/domain/contentmgmt/ContentProject.java

```diff
@@ -305,7 +305,7 @@ public void removeSource(ProjectSource source) {
     @Transient
     public List<PackageFilter> getPackageFilters() {
         return (List<PackageFilter>) getProjectFilters().stream()
```

Comment:
This first cast seems to be not needed with the new one below, should we remove it?